Obsidian/Net/Packets/Play/Clientbound/SetExperience.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Obsidian.Entities;
+using Obsidian.Serialization.Attributes;
+using System.Threading.Tasks;
+
+namespace Obsidian.Net.Packets.Play.Clientbound
+{
+    [ClientOnly]
+    public partial class SetExperience : ISerializablePacket
+    {
+        [Field(0)] public float ExperienceBar { get; set; }
+        
+        [VarLength] [Field(1)] public int Level { get; set; }
+        
+        [VarLength] [Field(2)] public int TotalExperience { get; set; }
+        public int Id => 0x48;
+        
+        public SetExperience(float experienceBar, int level, int totalExperience)
+        {
+            ExperienceBar = experienceBar;
+            Level = level;
+            TotalExperience = totalExperience;
+        }
+
+        public static SetExperience FromLevel(int level) => new (0, level, (int)GetExperience(level));
+        
+        private static float GetExperience(int level)
```

Comment:
This method shouldn't inside a packet class, instead inside a helper class or similar